src/TreeMap/TreeMapRect.tsx

```diff
@@ -96,6 +97,8 @@ export const TreeMapRect: FC<Partial<TreeMapRectProps>> = ({
     return getKey(data).join(' -> ');
   }, [data]);
 
+  const tooltipData = useMemo(() => ({ y: data.value, x: tooltipLabel }), [data, tooltipLabel]);
```

Comment:
Same https://github.com/reaviz/reaviz/pull/182/files#r1437926574

Reply:
This is not the same as https://github.com/reaviz/reaviz/pull/182/files#r1437926574. It's just moving the old tooltip value code into a variable for it to be reused.